test/scripts/install_chromedriver_travis.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
```

Comment:
Why do we need to install a different version of chrome?

Reply:
Default Chrome on Travis is of version 59 which is more likely to cause problems with chromedriver during tests. That's why I'm first making sure there are no chrome browsers on Travis, and then installing the latest stable one - this will be especially important when we move to running tests on Chrome in headless mode